### PR TITLE
Metadata configuration script

### LINKED ISSUE #107: fvar instances in sources (Design Space)
Here's a proposal of the fvar instances. After we reach the decision, we should implement the in the sources. 

Objective:
Simple and minimal approach to create a good user experience (avoid long drop-down lists). We're adding instances only for the weight axis. 
We're keeping the instances and their values compatible with first Google Sans, then for the new addition to the design space with Roboto Flex.

Details:
| Thin | **wght=100.0**, wdth=100.0, opsz=18.0, ROND=0.0, GRAD=0.0 |
| ExtraLight | **wght=200.0**, wdth=100.0, opsz=18.0, ROND=0.0, GRAD=0.0 |
| Light | **wght=300.0**, wdth=100.0, opsz=18.0, ROND=0.0, GRAD=0.0 |
| Regular | **wght=400.0**, wdth=100.0, opsz=18.0, ROND=0.0, GRAD=0.0 |
| Medium | **wght=500.0**, wdth=100.0, opsz=18.0, ROND=0.0, GRAD=0.0 |
| SemiBold | **wght=600.0**, wdth=100.0, opsz=18.0, ROND=0.0, GRAD=0.0 |
| Bold | **wght=700.0**, wdth=100.0, opsz=18.0, ROND=0.0, GRAD=0.0 |
| ExtraBold | **wght=800.0**, wdth=100.0, opsz=18.0, ROND=0.0, GRAD=0.0 |
| Black | **wght=900.0**, wdth=100.0, opsz=18.0, ROND=0.0, GRAD=0.0 |

https://github.com/googlefonts/googlesans-flex/issues/57#issuecomment-1334046326

### LINKED ISSUE #120: fix family and style max length
Font Bakery check fails here:

>> Check:   com.google.fonts/check/name/family_and_style_max_length
    Desc:    Combined length of family and style must not exceed 27 characters.
    Files:   GoogleSansFlex[GRAD,ROND,opsz,wdth,wght].ttf

     WARN The combined length of family and style exceeds 27                    
          chars in the following 'WINDOWS' entries: FONT_FAMILY_NAME =          
          'GoogleSansFlex opsz18-wdth100-wght400-ROND0' / SUBFAMILY_NAME =      
          'Regular'                                                             
                                                                                
          Please take a look at the conversation at                             
          https://github.com/googlefonts/fontbakery/issues/2179 in order to     
          understand the reasoning behind these name table records max-length   
          criteria. [code: too-long]                                            
    Result: WARN

The report is [here](https://github.com/googlefonts/googlesans-flex/actions/runs/3707745278/jobs/6284525053#step:9:284)
CC @chrissimpkins

### LINKED ISSUE #125: Add basic name table data
Here's the spec:
Platform ID3: 
ID0 (Copyright notice): "Copyright 2015 Google LLC. All Rights Reserved."
ID1 (Font Family name): "Google Sans Flex"
ID2 (Font Subfamily name): "Regular"
ID3 (Unique font identifier): "1.000;GOOG;GoogleSansFlex-Regular"
ID4 (Full font name ): "Google Sans Flex Regular"
ID5 (Version string): "Version 1.000"
ID6 (PostScript name): "GoogleSansFlex-Regular"
ID7 (Trademark): "Google Sans is a trademark of Google."
ID8 (Manufacturer Name): "Google LLC"
ID9 (Designer): "Google Sans Authors"
ID11 (URL Vendor): "https://design.google"
ID13 (License Description): "Google offers many fonts on open source terms. Google Sans Flex is not one of them. Please see google.com/fonts for alternatives."

This spec excludes instance and axis names (IDs 256+), and friendly feature names.

### PR DESCRIPTION
**Pull request by madig**
Add a script that modifies sources to implement 

* https://github.com/googlefonts/googlesans-flex/issues/125
* https://github.com/googlefonts/googlesans-flex/issues/107
* https://github.com/googlefonts/googlesans-flex/issues/120

### CODE REVIEW

File: sources/config.yaml [hunk 1]
```diff
@@ -81,12 +81,6 @@ stat:
 - name: Optical size
   tag: opsz
   values:
-  - flags: 0
```
Comment (chrissimpkins) [inline sources/config.yaml:84]: How is config.yaml playing into the STAT table build?

Comment (RickyDaMa) [reply sources/config.yaml:84]: It's read by `gftools`, which coordinates building the TTF for us currently

Comment (chrissimpkins) [reply sources/config.yaml:84]: Thanks! And it builds STAT with statmake or directly with fonttools, or some other approach?

Comment (madig) [reply sources/config.yaml:84]: The latter I believe, calling into axisregistry iirc.

**Comment by RickyDaMa** [comment]
As of the latest commit, I'm marking this as ready to review as I'm confident it solves the three linked issues

Example locally built fonts: [fonts.zip](https://github.com/googlefonts/googlesans-flex/files/10320418/fonts.zip)

Given the autonomous nature of this script, it could easily be added to the import workflow, save needing to be run manually

**Comment by RickyDaMa** [comment]
Note: the CI failure will be fixed by #146

**Pull request merged**